microservices-connector/config/manifests/llm-uservice_vllm.yaml

```diff
@@ -0,0 +1,282 @@
+---
+# Source: llm-uservice/charts/vllm/templates/configmap.yaml
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: llm-uservice-vllm-config
+  labels:
+    helm.sh/chart: vllm-1.0.0
+    app.kubernetes.io/name: vllm
+    app.kubernetes.io/instance: llm-uservice
+    app.kubernetes.io/version: "0.5"
+    app.kubernetes.io/managed-by: Helm
+data:
+  HF_TOKEN: "insert-your-huggingface-token-here"
+  http_proxy: ""
+  https_proxy: ""
+  no_proxy: ""
+  HABANA_LOGS: "/tmp/habana_logs"
```

Comment:
is this a gaudi service or just a superfluous here?

Reply:
oops, it's mistake to generate the manifest for CI related settings.